src/App.tsx

```diff
@@ -15,48 +15,94 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+enum SortType {
+  NONE,
+  ALPHABET,
+  LENGTH,
+}
+
 export const App: React.FC = () => {
+  const [sortType, setSortType] = useState(SortType.NONE);
+  const [isReversed, setIsReversed] = useState(false);
+  const [visibleGoods, setVisibleGoods] = useState(goodsFromServer);
```

Comment:
please add types to `useState` -> `useState<type>(initialValue)` 
example: `const [isReversed, setIsReversed] = useState<boolean>(false);`